terraform/concourse/elb.tf

```diff
@@ -1,3 +1,13 @@
+resource "aws_iam_server_certificate" "concourse" {
+  name = "${var.env}-concourse"
+  certificate_body = "${file("concourse.crt")}"
+  private_key = "${file("concourse.key")}"
+
+  provisioner "local-exec" {
+    command = "sleep 10"
```

Comment:
From the log in the commit message:
```
* aws_elb.concourse: [WARN] Error creating ELB Listener with SSL Cert, retrying: CertificateNotFound: Server Certificate not found for the key: arn:aws:iam::xxxx:server-certificate/xxxx
	status code: 400, request id: xxxx
```
Am I right in thinking that the Terraform run doesn't fail (non-zero exit code) when run without the `sleep`? It's a warning, rather than an error, which indicates that it had to be retried. If that's the case then I don't think we should add the `sleep` to mask the behaviour.